analyzers/src/SonarAnalyzer.CSharp/Rules/EquatableClassShouldBeSealed.cs

```diff
@@ -86,19 +84,17 @@ private static bool HasAnyInvalidIEquatableEqualsMethod(INamedTypeSymbol classSy
             var unprocessedTypeNames = equalsMethodsByTypeName.Keys.Except(equatableInterfacesByTypeName.Keys);
             return unprocessedTypeNames.Any(typeName => !equalsMethodsByTypeName[typeName].IsVirtual);
         }
-        private static bool IsCompilableIEquatableTSymbol(INamedTypeSymbol namedTypeSymbol)
-        {
-            return namedTypeSymbol.ConstructedFrom.Is(KnownType.System_IEquatable_T) &&
-                namedTypeSymbol.TypeArguments.Length == 1;
-        }
 
-        private static bool IsIEquatableEqualsMethodCandidate(IMethodSymbol methodSymbol)
-        {
-            return methodSymbol.MethodKind == MethodKind.Ordinary &&
-                methodSymbol.Name == EqualsMethodName &&
-                !methodSymbol.IsOverride &&
-                methodSymbol.ReturnType.Is(KnownType.System_Boolean) &&
-                methodSymbol.Parameters.Length == 1;
-        }
+        private static bool IsCompilableIEquatableTSymbol(INamedTypeSymbol namedTypeSymbol) =>
+            namedTypeSymbol.ConstructedFrom.Is(KnownType.System_IEquatable_T) &&
```

Comment:
nitpick: According to our coding style the operators should be at the beginning of the line (same for the method below).